### ISSUE TITLE
Mijn berichten - detailpagina - betalen link not focusable

### ISSUE THREAD

**Issue by TessaViergever** [nl-design-system/mijn-services#308]
“Betalen” link (looks like button) is niet focusable (⚠️ dit is een violation)

<img width="883" height="110" alt="Image" src="https://github.com/user-attachments/assets/f207d599-0c66-49a6-9cdb-87a8b7395a85" />
